### ISSUE TITLE
Incorrect path to favicon.ico

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#688]
(Original issue 981 created by arfrever on 2014-05-02T13:25:39.825112+00:00)

```
$ cd doc
$ make html
...
WARNING: favicon file 'favicon.ico' does not exist
...
```

Fix:
```
--- doc/conf.py
+++ doc/conf.py
@@ -117,7 +117,7 @@
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
-html_favicon = 'favicon.ico'
+html_favicon = '_static/favicon.ico'
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
```

**Comment by Anteru** [comment]
(Original issue 981 last updated on 2014-10-07T12:58:12.664469+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-10-07T12:58:12.679857+00:00)

Meanwhile fixed in 102ed2526c38.
